xsl/templates.xsl

```diff
@@ -184,8 +184,18 @@ SOFTWARE.
     </code>
   </xsl:template>
   <xsl:template name="par">
-    <xsl:param name="text"/>
-    <xsl:value-of select="$text"/>
+    <xsl:param name="text" />
+    <xsl:analyze-string select="$text" regex="(§[0-9]*)">
+      <xsl:matching-substring>
+        <xsl:variable select="regex-group(1)" name="paragraph" />
+        <a href="{concat('http://www.zerocracy.com/policy.html#', substring-after($paragraph,'§'))}">
```

Comment:
@paulodamaso We should use ``https://`` here, I see it is enabled for zerocracy.com, but redirection from ``http://`` to ``https://`` is not :(